ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -345,6 +351,70 @@ private void onAccountChangeMessage(String topic, String message) {
     }
   }
 
+  /**
+   * Selects {@link Container}s to be marked as INACTIVE. Check the valid data size of each DELETE_IN_PROGRESS container
+   * from {@link StatsSnapshot} and select the ones with zero data size to be marked as INACTIVE.
+   */
+  Set<Container> selectInvalidContainers(StatsSnapshot statsSnapshot) {
+    Map<String, Set<String>> accountToContainerMap = new HashMap<>();
+    Set<Container> invalidContainerSet = new HashSet<>();
+    getValidContainers(accountToContainerMap, statsSnapshot, null);
+    Set<Container> deprecatedContainerSet = getContainersByStatus(Container.ContainerStatus.DELETE_IN_PROGRESS);
```

Comment:
`deleteInProgressContianerSet`?

Reply:
I am a little bit confused about the naming here. 

In last PR, deprecated includes INACTIVE and DELETE_IN_PROGRESS , right?

Reply:
I've updated the name from deprecatedContainerSet to deleteInProgressContainerSet which include all containers marked as DELETE_IN_PROGRESS.